### PR TITLE
Update tool to version 3.5.1

### LINKED ISSUE #27: Add slevomat coding standard
Could it be possible to add the [Slevomat Coding Standard](https://github.com/slevomat/coding-standard#how-to-run-the-sniffs)'s rules to the available ruleset of PHP codesniffer ?
From the tool pattern list as well as from the configuration file.

I am currently using this standard trough a composer installation but this is of course not working in Codacy as it doesn't install the dependencies.

I thought that including it by default would be a good idea as it is a quite used standard and as this is what you did with angular plugin and eslint.

### PR DESCRIPTION
**Pull request by ljmf00**
Fixes #27, https://github.com/codacy/codacy-meta/issues/85 and https://github.com/codacy/codacy-meta/issues/138 .

This PR:
- Updated code to Scala 2.13
- Added slevomat coding standard plugin
- Updated the tool to 3.5.1
- Update PHP Security Audit to 2.0.1
- Update PHP Compatibility to 9.3.2
- Update Wordpress conding standards to 2.1.1
- Added Magento 2.x coding standards
- Update Magento 1.x coding standards to 4.0.0
- Fixed unnamed patterns and improve some patterns title
- Updated the Circle CI config to use Orbs

I was forced to do everything on the same PR because of inner dependency updates.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,59 +1,52 @@
...
-              docker login -u $DOCKER_USER -p $DOCKER_PASS
-              docker tag $CIRCLE_PROJECT_REPONAME codacy/$CIRCLE_PROJECT_REPONAME
-              docker tag $CIRCLE_PROJECT_REPONAME codacy/$CIRCLE_PROJECT_REPONAME:1.0.$CIRCLE_BUILD_NUM
-              docker push codacy/$CIRCLE_PROJECT_REPONAME
-            fi
+version: 2.1
+
+orbs:
+  codacy: codacy/base@1.0.0
+  codacy_plugins_test: codacy/plugins-test@0.3.1
```
Comment (lolgab) [inline .circleci/config.yml]: Can you bump this? The latest version is `0.4.0
`

Comment (ljmf00) [reply .circleci/config.yml]: Done

File: build.sbt [hunk 2]
```diff
@@ -7,15 +7,16 @@ name := """codacy-codesniffer"""
 
 version := "1.0-SNAPSHOT"
 
-val languageVersion = "2.12.7"
+val languageVersion = "2.13.1"
 
 scalaVersion := languageVersion
 
-resolvers ++= Seq("Typesafe Repo".at("http://repo.typesafe.com/typesafe/releases/"),
+resolvers ++= Seq("Typesafe Repo".at("https://repo.typesafe.com/typesafe/releases/"),
```
Comment (lolgab) [inline build.sbt]: You can remove resolvers here and use the default ones

Comment (ljmf00) [reply build.sbt]: done

**Comment by lolgab** [comment]
Very good job @ljmf00 👏

**Pull request merged**